pyproject.toml

```diff
@@ -20,8 +18,7 @@ include = [
 
 [tool.poetry.dependencies]
 python = "^3.8"
-poetry = "^1.6.0"
-poetry-core = "^1.7.0"
+poetry = "^1.8.0"
```

Comment:
this plugin imports directly from poetry-core, so it should explicitly depend on it

Reply:
Ahh, I was trying to find the import but didn't see it, let me fix that.

Reply:
Fixed